Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -459,7 +470,7 @@ extension ActivityStreamPanel: DataObserverDelegate {
     // Reloads both highlights and top sites data from their respective caches. Does not invalidate the cache.
     // See ActivityStreamDataObserver for invalidation logic.
     func reloadAll() {
-        accumulate([self.getHighlights, self.getTopSites]).uponQueue(.main) { _ in
+        accumulate([self.getHighlights, self.getTopSites, self.getPocketSites]).uponQueue(.main) { _ in
```

Comment:
If the network is slow. Then this will prevent topsites/history from loading. This should be done outside of this accumulate.